cms/envs/test.py

```diff
@@ -320,7 +320,3 @@
 ######### custom courses #########
 INSTALLED_APPS += ('openedx.core.djangoapps.ccxcon',)
 FEATURES['CUSTOM_COURSES_EDX'] = True
-
-# API access management. Necessary so that django-simple-history
-# doesn't break when running pre-test migrations.
-INSTALLED_APPS += ('openedx.core.djangoapps.api_admin',)
```

Comment:
Reviewers: If you have any ideas for how to prevent `django-simple-history` from running migrations for apps under `openedx` which aren't in `INSTALLED_APPS`, let me know. That would be preferable to duplicating every setting related to API management, which is necessary if `api_admin` is going to run under CMS settings.